shardingsphere-infra/shardingsphere-infra-authority/shardingsphere-infra-authority-common/src/main/java/org/apache/shardingsphere/authority/checker/AuthorityChecker.java

```diff
@@ -56,7 +58,28 @@ public SQLCheckResult check(final SQLStatement sqlStatement, final List<Object>
         // TODO add error msg
         return privileges.map(optional -> new SQLCheckResult(optional.hasPrivileges(Collections.singletonList(getPrivilege(sqlStatement))), "")).orElseGet(() -> new SQLCheckResult(false, ""));
     }
-    
+
+    @Override
+    public boolean check(final Grantee grantee, final AuthorityRule authorityRule) {
+        Optional<ShardingSphereUser> user = authorityRule.findUser(grantee);
+        if (!user.isPresent()) {
+            return false;
+        }
+        return true;
+    }
+
+    @Override
+    public boolean check(final Grantee grantee, final BiPredicate<Object, Object> validate, final Object cipher, final AuthorityRule authorityRule) {
```

Comment:
Maybe the first argument of `validate` is possible to be `password` rather than`grantee`?

Reply:
> Maybe the first argument of validate is possible to be password rather thangrantee?

This would be better, but it would cause MySQL and PostgreSQL to be incompatible. so I extract the password validation rule.

Reply:
TODO-4,
The third argument is likely to be `BiPredicate<Grantee, byte[]>` rather than `BiPredicate<Object, Object>`.  Thus, the argument `Object cipher` is supposed to be `byte[] cipher`.